### PR TITLE
06-Recursive Max Value Finder is ready to be reviewed.

### PR DESCRIPTION
**Pull request by csizmadiazsolt**
Please review it!

### CODE REVIEW

**Comment by ZsoltFabok** [comment]
Use proper pull request names, and give my id in the comments ( @ZsoltFabok) if you want to trigger a notification or show that something is ready to review ;-)

File: RecursiveFindMaximumValue/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Sat Jul 09 20:47:25 CEST 2016
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.9-bin.zip
```
Comment (ZsoltFabok) [inline RecursiveFindMaximumValue/gradle/wrapper/gradle-wrapper.properties]: Use 2.14

**Comment by ZsoltFabok** [comment]
Unfortunately, the exercise's description wasn't good enough. Sorry for that. You should take the structure as a string, convert it to lists and find the max.

As an alternative you can check how a flatMap works and see what difference does it make to find the desired value.

File: RecursiveFindMaximumValue/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (csizmadiazsolt) [inline RecursiveFindMaximumValue/gradle/wrapper/gradle-wrapper.properties]: What would be a good pull request name?

**Pull request closed without merging**